sdk/storage/azure-storage-blob/setup.py

```diff
@@ -94,7 +94,8 @@
     install_requires=[
         "azure-core<2.0.0,>=1.2.2",
         "msrest>=0.6.10",
-        "cryptography>=2.1.4"
+        "cryptography>=2.1.4",
+        "avro-python3"
```

Comment:
We can't use this as a dependency - it's python3 only.

Reply:
it's just a temp change, at that time internal avro parser wasn't ready